lib/bot.rb

```diff
@@ -49,6 +52,10 @@ def handle_pull_request(action:, title:, pr_number:)
   end
   # rubocop:enable Metrics/CyclomaticComplexity
 
+  def handle_pull_request_without_jira_issue
+    create_issue_and_update_github_pr_title if depfu_pr?
```

Comment:
Why only for depfu PRs?
If I have a small change I usually create & push to a new branch and use the QA comment to create a new ticket for it.
It would be nice if this could also be automated.

Reply:
I wanted to solve the issue with Depfu where it's definitely forgotten often.
I thought people were fine with adding the QA comments to regular PRs, after all you need one anyway?